### PR TITLE
🔨 Refactor some tests

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
# Description

Removes create venv

### CODE REVIEW

File: otx/cli/utils/tests.py [hunk 1]
```diff
@@ -99,13 +64,13 @@ def otx_train_testing(template, root, otx_dir, args):
...
     command_line.extend(args["train_params"])
-    assert run(command_line, env=collect_env_vars(work_dir)).returncode == 0
+    assert run(command_line).returncode == 0
     assert os.path.exists(f"{template_work_dir}/trained_{template.model_template_id}/weights.pth")
     assert os.path.exists(f"{template_work_dir}/trained_{template.model_template_id}/label_schema.json")
 
 
 def otx_hpo_testing(template, root, otx_dir, args):
-    work_dir, template_work_dir, _ = get_some_vars(template, root)
+    _, template_work_dir, _ = get_some_vars(template, root)
```
Comment (samet-akcay) [inline otx/cli/utils/tests.py]: Should we rename `get_some_vars` as `get_template_dir` or something?

Comment (samet-akcay) [reply otx/cli/utils/tests.py]: And perhaps only return `template_work_dir`?

Comment (ashwinvaidya17) [reply otx/cli/utils/tests.py]: Yeah makes sense!

File: tests/integration/cli/anomaly/test_anomaly_detection.py [hunk 2]
```diff
@@ -15,13 +15,13 @@
 # and limitations under the License.
```
Comment (samet-akcay) [inline tests/integration/cli/anomaly/test_anomaly_detection.py:15]: Should we make these changes to classification and segmentation as well?

Comment (ashwinvaidya17) [reply tests/integration/cli/anomaly/test_anomaly_detection.py:15]: Oh right I forgot about those 😅 I was just testing detection

**Pull request merged**